src/Routes/RosSuggestedInstanceTypes/RosSuggestedInstanceTypes.js

```diff
@@ -35,15 +35,17 @@ const SuggestedInstance = () => {
                         <PageHeaderTitle title='Suggested Instance Types'/>
                     </PageHeader>
                     {
-                        hasSuggestedInstances   //TODO: remove or update this check while working on RHIROS-1222
-                            ? <SuggestedInstanceTypesTable />
-                            : systemCount !== 0
-                                ? <NoEntitiesFound />
-                                : (
-                                    <Bullseye>
-                                        <Spinner />
-                                    </Bullseye>
-                                )
+                        !loading ?
```

Comment:
@r14chandra can you please also handle errors here? I guess ATM in case of errors it shows spinner